fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/PredicateMultiMap.java

```diff
@@ -61,6 +62,9 @@ public interface CompensatePredicateFunction {
                 (partialMatch, boundPrefixMap) -> {
                     throw new RecordCoreException("should not be called");
                 };
+
+        CompensatePredicateFunction EMPTY =
```

Comment:
Maybe make this `private` and add a new static method returning this `empty()`

Reply:
Since it is an interface, we can't hide it. But I added a static method `empty()` and I use it throughout the codebase instead. Thanks for the suggestion.

Reply:
Thank you!